src/App.vue

```diff
@@ -11,22 +11,28 @@
 </template>
 
 <script>
-    import EmptyLayout from '@/layouts/EmptyLayout.vue'
-    import MainLayout from '@/layouts/MainLayout.vue'
-
-    export default {
-
-        computed: {
-
-            layout() {
-                return this.$route.meta.layout
-            },
-        },
-
-        components: {
-            EmptyLayout, MainLayout
-        }
+  import EmptyLayout from '@/layouts/EmptyLayout.vue'
+  import MainLayout from '@/layouts/MainLayout.vue'
+  import {mapActions} from "vuex"
+
+  export default {
+    computed: {
+      layout() {
+        return this.$route.meta.layout
+      },
+    },
+    components: {
+      EmptyLayout, MainLayout
+    },
+    methods: {
+      ...mapActions({
+        setClearState: 'callSetClearState_actions',
+      }),
+    },
+    async beforeMount() {
```

Comment:
when you have only one await method in async func you don't need to add async/await